lib/Client.js

```diff
@@ -1245,7 +1245,7 @@ class Client extends EventEmitter {
     }
 
     /**
-    * Edit the bot's nickname in a guild
+    * [DEPRECATED] Edit the bot's nickname in a guild
```

Comment:
is it? i don't think it's DEPRECATED as per discord docs
https://discord.com/developers/docs/resources/guild#modify-guild-member

Reply:
Pretty sure he's just deprecating Client#editNickname, not nicknames as a whole

Reply:
That's because the PR for it hasn't been merged yet. Same reason why the documentation for per guild avatars also doesn't exist.

Reply:
@DonovanDMC then i'll recommend mentioning that to avoid confusing users.

Reply:
It's fine as it is

Reply:
> @DonovanDMC then i'll recommend mentioning that to avoid confusing users.
I'd argue most users won't assume nickname editing as a whole isn't getting deprecated

You also linked the wrong docs section. The one this references is one below that,
[https://discord.com/developers/docs/resources/guild#modify-current-user-nick](https://discord.com/developers/docs/resources/guild#modify-current-user-nick)